core/tabs/utils/numlock.sh

```diff
@@ -48,7 +48,7 @@ numlockSetup() {
     create_service
   fi
 
-  printf "%b\n" "Do you want to enable Numlock on boot? (y/n): "
+  printf "%b\n" "Do you want to enable Numlock on boot? (Y/n): "
   read -r confirm
   if [ "$confirm" = "y" ] || [ "$confirm" = "Y" ]; then
     "$ESCALATION_TOOL" systemctl enable numlock.service --quiet
```

Comment:
y is not the default here. Shouldn't be `Y`.

```suggestion
  printf "%b\n" "Do you want to enable Numlock on boot? (y/N): "
  read -r confirm
  if [ "$confirm" = "y" ] || [ "$confirm" = "Y" ]; then
    "$ESCALATION_TOOL" systemctl enable numlock.service --quiet
```

Reply:
remove the \n aswell

Reply:
Done